tools/bazel.rc

```diff
@@ -76,7 +76,10 @@ build --copt="-Werror=return-type"
 build --copt="-Werror=unused-variable"
 build --copt="-Werror=unused-but-set-variable"
 build --copt="-Werror=switch"
-build --cxxopt="-Werror=sign-compare"
+# TODO(changh726): disable the following line temporarily as
+# modules/dreamview/backend/simulation_world/ \
+# simulation_world_service.cc:224: can't compile
+#build --cxxopt="-Werror=sign-compare"
```

Comment:
> Seems an easy issue. Please fix the corresponding code instead of turning off global check.
> If it's also in master, fix there, too.

Agree. The `-Werror` copts should be fixed in  `docker_dev` branch and submitted to the master branch as long as it doesn't break compilation there.